lightning/src/ln/peers/conduit.rs

```diff
@@ -0,0 +1,173 @@
+use ln::peers::{chacha, hkdf};
+
+/// Returned after a successful handshake to encrypt and decrypt communication with peer nodes
```

Comment:
Can you add more details about how you'd use this, what the requirements are for use, etc? Same goes for pretty much everything pub in this PR.